options/ansi/generic/wchar-stubs.cpp

```diff
@@ -190,6 +189,8 @@ size_t wcsrtombs(char *mbs, const wchar_t **wcsp, size_t mb_limit, mbstate_t *st
 		return size;
 	}
 
+	mlibc::code_seq<char> nseq{mbs, mbs + mb_limit};
```

Comment:
Does this also need the `frg::min(..., MB_CUR_MAX)` like in `stdlib-stubs.cpp`?

Reply:
No, as for `wcsrtombs` the limit is purely given by the `mb_limit` function parameter.